web/src/components/profile/Profile.tsx

```diff
@@ -75,9 +76,11 @@ export const Profile: FC = () => {
             currentSubject={sessionsMe?.auth_sub}
           />
           <ProfileReferalLinks />
+          {user.bitzlato_user && <ChangePassword email={user.bitzlato_user?.email} />}
```

Comment:
```suggestion
          {user.bitzlato_user && <ChangePassword email={user.bitzlato_user.email} />}
```
У тебя и так проверка есть